### PR TITLE
[warm reboot] save configuration after warm reboot

### PR DESCRIPTION
**Pull request by yxieca**
**- What I did**

After warm reboot, save a copy of in memory database to config_db.json,
upgrade procedure might have removed config_db.json to force new image
to reload minigraph. However, reload minigraph is skipped during warm
reboot. Missing config_db.json would cause device to fault in next
non-upgrading cold/fast reboot.

Signed-off-by: Ying Xie <ying.xie@microsoft.com>

### CODE REVIEW

**Comment by yxieca** [comment]
retest this please

**Comment by yxieca** [comment]
retest this please

File: files/image_config/warmboot-finalizer/finalize-warmboot.sh [hunk 1]
```diff
@@ -78,6 +78,11 @@ if [[ x"${WARM_BOOT}" != x"true" ]]; then
     exit 0
 fi
 
+# No need to wait for the reconciliation process. Database has been loaded
+# and migrated. This is good enough to save a copy.
+debug "Save in memory database after warm reboot ..."
```
Comment (jleveque) [inline files/image_config/warmboot-finalizer/finalize-warmboot.sh]: Suggest adding a hyphen: `in-memory`

Comment (yxieca) [reply files/image_config/warmboot-finalizer/finalize-warmboot.sh]: Done

**Pull request merged**